File: pec-observation-app/android-app/app/src/main/kotlin/br/gov/educacao/sp/pecobservation/domain/entities/School.kt
package br.gov.educacao.sp.pecobservation.domain.entities

import java.time.Instant
import java.util.UUID

data class School(
    val id: UUID = UUID.randomUUID(),
    val name: String,
    val inepCode: String,
    val address: String = "",
    val city: String = "",
    val createdAt: Instant = Instant.now(),
)

File: pec-observation-app/web-app/src/domain/entities/School.ts
export interface School {
  id: string
  inepCode: string
  name: string
  city: string
  state: string
  address?: string
  principalName?: string
  isActive: boolean
  createdAt: string
}

export interface Teacher {
  id: string
  schoolId: string
  name: string
  email: string
  subjectArea: string
  employeeId?: string
  isActive: boolean
  createdAt: string
}
